src/components/form-elements/TextInput.jsx

```diff
@@ -41,7 +41,8 @@ const TextInput = props => {
     'sg-input--invalid': valid === VALIDATION.INVALID,
     'sg-input--full-width': fullWidth,
     'sg-input--no-border': noBorder,
-    'sg-input--with-icon': withIcon
+    'sg-input--with-icon': withIcon,
+    'sg-search__input': type === 'search'
```

Comment:
later we can move it to Search component (like rwd helper) or if there is more cases we can extend search component for more cssClasses.
Pls create issue in GitHub